boefjes/boefjes/plugins/kat_rpki/normalize.py

```diff
@@ -10,14 +11,18 @@ def run(input_ooi: dict, raw: bytes) -> Iterable[NormalizerOutput]:
     results = json.loads(raw)
     ooi = Reference.from_str(input_ooi["primary_key"])
 
-    if not results["exists"]:
-        ft = KATFindingType(id="KAT-NO-RPKI")
-        f = Finding(finding_type=ft.reference, ooi=ooi)
-        yield ft
-        yield f
+    address = IPv4Address(ooi.tokenized.address)
 
-    if not results.get("valid") and not results.get("notexpired"):
-        ft = KATFindingType(id="KAT-EXPIRED-RPKI")
-        f = Finding(finding_type=ft.reference, ooi=ooi)
-        yield ft
-        yield f
+    # if the address is private, we do not need a ROA
+    if not address.is_private:
```

Comment:
```suggestion
    if address.is_global:
```
not is_private fails for 100.64.0.0/10, which is not global but also not technically private.

It's also sementically more correct, there is only RPKI on global IP space :)

Both cases would also fail for globally addressable IP space that is not publically routed, like 7.0.0.0/8 (US Department of Defense).  But that's probably not significant among users of openkat :)